tests/test_deployment.py

```diff
@@ -19,7 +20,12 @@ def test_e2e_deployment(new_deployment_pg, variables):
     actions performed on each page and the yaml values that direct those
     actions are handled by the UIDeploymentRunner class.
     '''
-    runner = UIDeploymentRunner()
+
+    # First parse the config and instantiate the deployment runner
+    config = DeploymentConfig()
```

Comment:
This should be a global pytest fixture that any test can reference so that it's initialization is standard for any tests that won't use the deployment runner like api tests.

Initialization should also read the variables filename that is passed on the pytest command line (--variables) or just use the "variables" fixture object to read the config instead of re-reading the file manually